tests/features/targets/import_targets.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Import targets directly to RSTUF SQL Database
+    As an admin,
+    Admin has deployed and bootstrapped RSTUF
+    Admin has an existent large number of targets (i.e.: 1,000,000 targets)
+    Admin wants to load targets in chunks to process them faster
+
+    Background:
+        Given admin has the RSTUF CLI in a host with access to RSTUF SQL Database
+        And admin has logged in with RSTUF CLI
+        And admin has the credentials to access RSTUF SQL Database
+        And admin has exported the targets info to CSV files (respecting CSV limits) with the format 'path;size;hash_type;hash;custom;'
+
+    Scenario: Admin imports a large number of targets to RSTUF
+        Given admin has multiple CSV files with targets 'tests/data/targets-1of2.csv' and 'tests/data/targets-2of2.csv'
+        When admin runs 'rstuf -c rstuf.ini admin import-targets -csv tests/data/targets-1of2.csv -csv tests/data/targets-2of2.csv -metadata-url http://127.0.0.1:8080 -db-uri postgresql://postgres:secret@127.0.0.1:543'
```

Comment:
I see you directly pass both of the files in one command.
I see how that could be useful: write a command and don't think about the process for a couple of hours, but if I was responsible for the migration I am sure I will validate that the loading of each of the chunks was successful after each manual import. 
We can support of course multiple and single file targets, but I just wanted to mention that.
Probably it's not relevant to the feature file as we want to showcase we support importing multiple files at once.

Reply:
> I see you directly pass both of the files in one command.
> I see how that could be useful: write a command and don't think about the process for a couple of hours, but if I was responsible for the migration I am sure I will validate that the loading of each of the chunks was successful after each manual import.

The process in the CLI can take a long time to finalize but is not a problem to interrupt after the task id is generated. It will be running in the background and the user could check the task status.

> We can support of course multiple and single file targets, but I just wanted to mention that.

Actually, it was my first approach. But the computation time is very big.
Importing the chunks of 500,000 targets takes ~15s, and publishing them takes approximately 40min --  all these numbers are based on my laptop [as mentioned here](https://github.com/vmware/repository-service-tuf/issues/188#issuecomment-1400576033).
Loading 1,000,000 targets in the DB first (it imports by chunks) will take ~30s and publishing all them  (1M) takes ~55 minutes.

> Probably it's not relevant to the feature file as we want to showcase we support importing multiple files at once.

It is a good comment, maybe if there are requests from uses, we can have an implementation that supports both ways of importing data.

Reply:
> It is a good comment, maybe if there are requests from uses, we can have an implementation that supports both ways of importing data.

Don't we support both ways right now?
You allow sending multiple CSV files and of course, you can send them one by one.

Reply:
Yes, that is correct.